ruby/src/zip_ruby_layer.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -e
+set -u
+
+echo_usage() {
+	echo "usage: Build Lambda layer/application by SAM"
+	echo " -n <specify layer name>"
+}
+
+is_sample() {
```

Comment:
do we need this function? 
Seem it is not referenced in this bash

Reply:
no, should just remove it.